docs-src/concepts/what-is-a-list-filter.md

```diff
@@ -31,19 +31,17 @@ A List Filter contains [Search Attribute](/concepts/what-is-a-search-attribute)
   - **BETWEEN ... AND**
   - **IN**
   - **ORDER BY**
-  - **LIKE**
-
-- Wildcards ('\*', '%', etc.) are supported for string-type List Filter Search Attributes.
-  Use wildcards with the **LIKE** operator to query possible values:
-
-  ```bash
-  // Create a "ProductId" custom Search Attribute of type String
-  tctl admin cluster add-search-attributes --name ProductId --type String
-  // Match values that start with "book"
-  ProductId LIKE "book%"
-  // Match values that contain "favorite"
-  ProductId LIKE "%favorite%"
-  ```
+
+  The last operator (**ORDER BY**) is supported only with Elasticsearch as the Visibility store.
+  Custom Search Attributes of `Text` type cannot be used in **ORDER BY** clauses.
+
+- To use a partial string match (i.e. specific word), create a [custom Search Attribute](/app-dev-context/observability#custom-search-attributes) of type `Text` and set it value to a phrase that contains that word.
+
+- The `=` operator works like **CONTAINS** to find Workflows with Search Attributes that contain a specific word.
```

Comment:
@DJSanti I would reorder these bullets and talk about how the `=` operator works first - then back that up with the use case of partial string matches - and the additional examples that Alex has - 

And make sure the information is properly nested.. right now - these bullets are at the same level - so someone would maybe get confused about whether the information was linear.

Reply:
Idea: it's reordered but not as a bulleted list.

Reply:
Idea: reorder these bullet points, but not as a list. Instead, write out the page some while keeping this information.